File: Data Structures & Algorithms/longest-common-prefix/submission-2.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        Arrays.sort(strs);
        String a=strs[0];
        String b=strs[strs.length-1];
        String ans="";
        int l=0;
        while(l<a.length() && l<b.length() && a.charAt(l)==b.charAt(l)){
            ans+=a.charAt(l);
            l++;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/merge-strings-alternately/submission-2.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        ans=""
        l=0
        r=0
        while l<len(word1) and r<len(word2):
            ans+=word1[l]
            ans+=word2[r]
            l+=1
            r+=1
        while l<len(word1):
            ans+=word1[l]
            l+=1
        while r<len(word2):
            ans+=word2[r]
            r+=1
        return ans
